### PR TITLE
Remove hard coded ballerina version in java code

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

## Remarks
https://github.com/ballerina-platform/ballerina-standard-library/issues/788

### CODE REVIEW

File: nats-native/src/main/java/org/ballerinalang/nats/Constants.java [hunk 1]
```diff
@@ -53,12 +52,10 @@
 
     public static final String NATS = "nats";
     public static final String ORG_NAME = "ballerinax";
-    public static final String VERSION = "1.0.7";
 
     // Represents nats package.
-    public static final String NATS_PACKAGE = ORG_NAME + ORG_NAME_SEPARATOR + NATS + ":" + VERSION;
-
-    public static final Module NATS_PACKAGE_ID = new Module(ORG_NAME, "nats", VERSION);
+    public static final String NATS_PACKAGE = ORG_NAME + ORG_NAME_SEPARATOR + NATS + ":"
```
Comment (ThisaruGuruge) [inline nats-native/src/main/java/org/ballerinalang/nats/Constants.java:57]: Why do we need this?

File: nats-native/src/main/java/org/ballerinalang/nats/Constants.java [hunk 2]
```diff
@@ -128,9 +125,11 @@
     public static final BString MAX_BYTES = StringUtils.fromString("maxBytes");
 
     // Strand meta data
-    public static final StrandMetadata ON_MESSAGE_METADATA = new StrandMetadata(ORG_NAME, NATS, VERSION,
+    public static final StrandMetadata ON_MESSAGE_METADATA = new StrandMetadata(ORG_NAME, NATS,
```
Comment (ThisaruGuruge) [inline nats-native/src/main/java/org/ballerinalang/nats/Constants.java:128]: We can create the StrandMetadata when we need it, and use the `getModule().getOrg()`, `getModule().getName()`, etc. to handle these.

File: nats-native/src/main/java/org/ballerinalang/nats/Utils.java [hunk 3]
```diff
@@ -37,8 +39,25 @@
...
+
+    public static void setModule(Environment env) {
+        natsModule = env.getCurrentModule();
+    }
+
+    public static Module getModule() {
+        return natsModule;
+    }
+
+    public static String getVersion() {
```
Comment (ThisaruGuruge) [inline nats-native/src/main/java/org/ballerinalang/nats/Utils.java:55]: Do we really need this separate method?

File: nats-native/src/main/java/org/ballerinalang/nats/Constants.java (hunk 1 above)
Comment (aashikam) [reply nats-native/src/main/java/org/ballerinalang/nats/Constants.java]: Fixed in c8e94613ed7ff23ec27267c9e3a2bb186a1d285e

File: nats-native/src/main/java/org/ballerinalang/nats/Constants.java (hunk 2 above)
Comment (aashikam) [reply nats-native/src/main/java/org/ballerinalang/nats/Constants.java:86]: Fixed in c8e94613ed7ff23ec27267c9e3a2bb186a1d285e

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**